kolibri/plugins/coach/assets/src/modules/groups/handlers.js

```diff
@@ -41,7 +17,7 @@ export function showGroupsPage(store, classId) {
   return ConditionalPromise.all(promises).only(
     samePageCheckGenerator(store),
     ([classUsers, groupsCollection]) => {
-      const groups = _groupsState(groupsCollection);
+      const groups = groupsCollection.map(group => ({ ...group, users: [] }));
```

Comment:
we can probably make this more efficient by leveraging data already available in the `classSummary` module rather than fetching from the collections.

Not necessary for now - will file a follow-up issue

Reply:
created follow-up issue: https://github.com/learningequality/kolibri/issues/4811